guava-gwt/src-super/com/google/common/collect/super/com/google/common/collect/Synchronized.java

```diff
@@ -1303,5 +1303,192 @@ public E remove() {
 
     private static final long serialVersionUID = 0;
   }
+  static <R, C, V> Table<R, C, V> table(final Table<R, C, V> table, @Nullable final Object mutex) {
+    return new SynchronizedTable<R, C, V>(table, mutex);
+  }
+
+  static class SynchronizedTable<R, C, V> extends SynchronizedObject implements Table<R, C, V>, Serializable {
+
+    SynchronizedTable(Table<? extends R, ? extends C, ? extends V> delegate, Object mutex) {
+      super(delegate, mutex);
+    }
+
+    @SuppressWarnings("unchecked") // safe, covariant cast
+    @Override
+    protected Table<R, C, V> delegate() {
+      return (Table<R, C, V>) delegate;
+    }
+
+    @Override
+    public Set<Table.Cell<R, C, V>> cellSet() {
+      synchronized (mutex) {
+        return set(delegate().cellSet(), mutex);
+      }
+    }
+
+    @Override
+    public void clear() {
+      synchronized (mutex) {
+        delegate().clear();
+      }
+    }
+
+    @Override
+    public Map<R, V> column(C columnKey) {
+      synchronized (mutex) {
+        return map(delegate().column(columnKey), mutex);
+      }
+    }
+
+    @Override
+    public Set<C> columnKeySet() {
+      synchronized (mutex) {
+        return set(delegate().columnKeySet(), mutex);
+      }
+    }
+
+    @Override
+    public Map<C, Map<R, V>> columnMap() {
+      synchronized (mutex) {
+        Function<Map<R, V>, Map<R, V>> mapFunction = synchronizedMapWrapper(mutex);
+          Map<C, Map<R, V>> columnMap = delegate().columnMap();
```

Comment:
Wait, I wasn't referring to indenting these; they should be at the same level as the rest of the code...